src/siteconfig/models.py

```diff
@@ -246,6 +246,12 @@ class SiteConfig(models.Model):
             or \"Skill\" might be a more suitable name, depending on how you use the Tags feature."
     )
 
+    custom_profile_field = models.CharField(
+        default="", blank=True, max_length=20,
+        help_text="This field can be used to collect custom data such as a student's grad year or other \
+            important info you need from your users. If left blank this field will be ignored."
```

Comment:
"...will be ignored and will not appear on student profiles."

Sorry, changed my ind and added a bit more, since I've request other changes anyway =D